examples/hybrid/cli_options.jl

```diff
@@ -151,7 +151,7 @@ function parse_commandline()
         arg_type = Int
         default = 10
         "--nh_poly"
-        help = "Horizontal polynomial order"
+        help = "Horizontal polynomial degree"
```

Comment:
```suggestion
        help = "Horizontal polynomial degree. Note: The number of quadrature points (or knots) within each element is then Nq = <--nh_poly> + 1."
```

Reply:
Thanks @akshaysridhar ! How about the proposed change in [[7794af]](https://github.com/CliMA/ClimaAtmos.jl/pull/860/commits/7794af2084180ad7d45bff85fca58c2f3790d4d6)? I added "by default" b/c we can in principle specify different Nq's to allow for under/over-integration